### DISCUSSION TITLE
blank template

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#533] [Ideas]
I have a suggestion.
I think it would be good if there was an empty template, without presets and without layout. This way you can build your own template from scratch without having leftovers from template Zero, One or Two.

**Comment by Chacapamac** [comment]
@Solero-Artaxerxes 
I think this is a good idea, a minimalist template. Astroid is so user-friendly at building whatever you think about. For People with a bit of experience it is easier to start new with the exact structure they want without having to deconstruct the template to rebuild on top.

I guess, you will have to keep at least a minimal menu and maybe a “Featured” or “Single Article” for the home page?
**How do you see it?**

**UNRELATED —** 😇
I am building a template that will show off Astroid-specific Super Powers. My first page, (Home), is built with only one article in the normal Joomla Main Content (with all the Joomla/Astroid abilities of a Main article, like Seo, Versioning, and Open Graph). This “Home” page article will be easy to construct with “sections” that resemble multiple template layout positions, making it hyper-easy to manage. For Sure, this comes with a Custom Layout that I name “ONE ARTICLE HOME PAGE” 

I should put this Demo online relatively soon to invite people to participate. The goal is to present a minimalist Demo that can show off Astroid and help Astroid users see how it's done. I also want to present all the necessary page types needed in a Modern website ( Blog, Contact....), and Astroid stuff like Megamenu, Page Banners, Blog and Articles Styles and more.  All that in an easy-to-learn and transform package.

**Comment by ghost** [comment]
@Chacapamac 

> I guess, you will have to keep at least a minimal menu and maybe a “Featured” or “Single Article” for the home page?

I had not thought about that. A minimal menu and a single article for the homepage would be good, you're right.
